.github/workflows/build.yml

```diff
@@ -315,16 +288,39 @@ jobs:
           password: ${{ secrets.ARTIFACTORY_TOKEN }}
 
       - name: Run Tests
-        run: mvn -B jar:jar jar:test-jar surefire:test -DtrimStackTrace=false -Dliquibase.sdk.testSystem.test=${{ matrix.testSystem }} -Dliquibase.sdk.testSystem.acceptLicenses=${{ matrix.testSystem }} -Dtest=*IntegrationTest,*ExecutorTest -DfailIfNoTests=false -Dsurefire.failIfNoSpecifiedTests=false
+        run: mvn -B -pl '!liquibase-cdi-jakarta' -P 'integration-test' clean verify -DtrimStackTrace=false -Dliquibase.sdk.testSystem.test=${{ matrix.testSystem }} -Dliquibase.sdk.testSystem.acceptLicenses=${{ matrix.testSystem }} -Dtest='*IntegrationTest,*ExecutorTest' -DfailIfNoTests=false -Dsurefire.failIfNoSpecifiedTests=false
```

Comment:
Any reason you're skipping liquibase-cdi-jakarta here? I'm just curious. I don't think it's a problem.

But, it would be great if you could put a comment in the build.yml here explaining why you're doing this. We're sure to forget by the next time we have to refactor this.

Reply:
corrected